content/rdi/quickstart/write-behind-guide.md

```diff
@@ -40,7 +40,7 @@ RDI write-behind currently supports these target data stores:
 
 ## Prerequisites
 
-The only prerequisite for running RDI write-behind is [Redis Gears Python](https://redis.com/modules/redis-gears/) >= 1.2.6 installed on the Redis Enterprise Cluster and enabled for the database you want to mirror to the downstream data store.
+The only prerequisite for running RDI write-behind is [Redis Gears Python](https://redis.com/modules/redis-gears/) >= 1.2.7 installed on the Redis Enterprise Cluster and enabled for the database you want to mirror to the downstream data store.
```

Comment:
We should use the variable in `config.yaml` instead of putting `1.2.7` here hardcoded.

Reply:
✅